bin/env.sh

```diff
@@ -14,7 +14,7 @@ if [ ! -d "${BUILDDIR}/${VENV}" ]; then
     PYTHON_MAJOR=$(${PYTHON} -c 'import sys; print(sys.version_info[0])')
     PYTHON_MINOR=$(${PYTHON} -c 'import sys; print(sys.version_info[1])')
 
-    if [ "${PYTHON_MAJOR}" != "3" ] || [ "${PYTHON_MINOR}" != "5" ]; then
+    if [ "${PYTHON_MAJOR}" != "3" ] || [[ "${PYTHON_MINOR}" < "5" ]]; then
```

Comment:
I wouldn't just make this change directly and commit it as is. This would allow anyone who's default python installation is 3.6 to unwittingly pass this check. We should only allow python 3.6 in conjunction with some sort of DCOS_CLI_EXPERIMENTAL variable being set.

Reply:
Good point, this syntax was also failing on winbash apparently. I'll look into it.